mediacapture-record/MediaRecorder-start.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html>
+<head>
+  <title>MediaRecorder Start</title>
+  <link rel="help" href="https://w3c.github.io/mediacapture-record/MediaRecorder.html#dom-mediarecorder-start">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<canvas id="canvas" width="200" height="200">
+</canvas>
+<script>
+    function createVideoStream() {
+        const canvas = document.getElementById("canvas");
```

Comment:
nit: this is unnecessary, elements with an id are already bound to a variable on the global scope with the value of the id attribute.

Reply:
Oh, I didn't know that. Is that something that comes from the `testharness.js` script? I will remove this line then.